controllers/urls.js

```diff
@@ -1,16 +1,51 @@
-const express = require('express')
-const Url = require('../models/Url')
+const express = require('express');
+const { validationResult } = require('express-validator');
+const statusCodes = require('http-status-code');
+const Url = require('../models/Url');
+const generateRandomString = require('../middleware/randomString');
 
-const createShortUrl = async(req, res) => {
-    const url = await Url.create({...req.body})
-    res.json(url)
-}
+const createShortUrl = async (req, res) => {
+  const { customName, originalUrl } = req.body;
+  let shortUrls;
 
-const customShortUrl = async(req, res)  => {
-    res.send('custom name for short url')
-}
+  try {
+    if (customName) {
+      const customNameWithDashes = customName.split(' ').join('-');
+      shortUrls = `https://shortit/${customNameWithDashes}`;
```

Comment:
Create a constant file and add this BASEURL so you can reuse it.

`const BASEURL = https://shortit`

```suggestion
      shortUrls = `${BASEURL}/${customNameWithDashes}`;
```

Reply:
done on 24e430e